Misc/NEWS.d/next/Library/2023-01-21-16-50-22.gh-issue-100795.NPMZf7.rst

```diff
@@ -0,0 +1,2 @@
+Avoid unexpected ``freeaddrinfo`` when :mod:`socket.socket.getaddrinfo`
```

Comment:
```suggestion
Avoid unexpected ``freeaddrinfo`` when :meth:`socket.socket.getaddrinfo`
```

Being in hurry, I've left a part from my earlier phrasing.

You can apply the suggestion above by clicking `Commit suggestion` button there.